pkg/controller/secret/secret_controller.go

```diff
@@ -410,14 +412,25 @@ func createWatchdogReceivers(watchdogURL string) []*alertmanager.Receiver {
 	}
 }
 
+// createHttpConfig creates a HttpConfig used for receivers that can accept that configuration
+func createHttpConfig(clusterProxy string) alertmanager.HttpConfig {
+	if clusterProxy == "" {
+		return alertmanager.HttpConfig{}
+	}
+	return alertmanager.HttpConfig{
+		ProxyURL:  clusterProxy,
+		TLSConfig: alertmanager.TLSConfig{},
```

Comment:
do we need to fetch the CA bundle for the proxy and populate it here?

Reply:
Based on my tests so far, as long as a cluster owner follows our procedure for adding their custom CA to the system CA truststore, it automatically gets included in the AM container's trusted CA bundle and no special handling needs to happen outside of that.

Reply:
To specifically address your question here, AlertManager has a ConfigMap `alertmanager-trusted-ca-bundle` which has the `inject-trusted-cabundle` label, therefore, when a custom CA Bundle is added to the cluster, it will automatically be added to the AlertManager CA bundle by the cluster network operator. Therefore no CA bundle should need to be specified in the AlertManager config.